utils/smoothing/src/lib.rs

```diff
@@ -34,7 +34,7 @@ pub fn extrapolated_cum_sum_of_ratio(
 
     let squared_velo_2 = (velo_2 * velo_2) >> PRECISION;
 
-    if squared_velo_2 >= get_cum_sum_ratio_epsilon() {
+    if squared_velo_2 >= EPSILON.clone() {
```

Comment:
when accessing the lazy static, you don't need to clone them, can just deref and/or reference them. Should only need to clone them if you need to initialize a new variable